### PR TITLE
VR-8150 move common source to separate dir

### PR DESCRIPTION
**Pull request by evgeniilezhnin**

### CODE REVIEW

File: backend/common/src/main/java/ai/verta/modeldb/common/CommonUtils.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import com.google.rpc.Status;
+import io.grpc.StatusRuntimeException;
+import io.grpc.protobuf.StatusProto;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class CommonUtils {
+  private static final Logger LOGGER = LogManager.getLogger(CommonUtils.class);
+
+  public static Integer getRegisteredBackgroundUtilsCount() {
```
Comment (anandjakhaniya) [inline backend/common/src/main/java/ai/verta/modeldb/common/CommonUtils.java:14]: If you have get this here then related all function like `setRegisteredBackgroundUtilsCount` etc should be here in common module.

Comment (evgeniilezhnin) [reply backend/common/src/main/java/ai/verta/modeldb/common/CommonUtils.java:14]: moved

File: backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java [hunk 2]
```diff
@@ -643,54 +646,21 @@ public static String getJoinedLocation(List<String> location) {
...
-          Status.newBuilder().setCode(Code.UNAVAILABLE_VALUE).setMessage(errorMessage).build();
-      throw StatusProto.toStatusRuntimeException(status);
-    }
-    throw ex;
-  }
-
   public static void initializeBackgroundUtilsCount() {
     int backgroundUtilsCount = 0;
     try {
-      if (System.getProperty(ModelDBConstants.BACKGROUND_UTILS_COUNT) == null) {
```
Comment (anandjakhaniya) [inline backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java:682]: this `initializeBackgroundUtilsCount` function should be in commin

File: backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java [hunk 3]
```diff
@@ -700,37 +670,24 @@ public static void initializeBackgroundUtilsCount() {
    */
   public static void registeredBackgroundUtilsCount() {
```
Comment (anandjakhaniya) [inline backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java]: `registeredBackgroundUtilsCount` should function should be in common

File: backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java [hunk 4]
```diff
@@ -700,37 +670,24 @@ public static void initializeBackgroundUtilsCount() {
...
+      backgroundUtilsCount = CommonUtils.getRegisteredBackgroundUtilsCount();
     }
     backgroundUtilsCount = backgroundUtilsCount + 1;
     LOGGER.trace("After registered runningBackgroundUtilsCount : {}", backgroundUtilsCount);
     System.setProperty(
-        ModelDBConstants.BACKGROUND_UTILS_COUNT, Integer.toString(backgroundUtilsCount));
+        CommonConstants.BACKGROUND_UTILS_COUNT, Integer.toString(backgroundUtilsCount));
   }
 
   public static void unregisteredBackgroundUtilsCount() {
```
Comment (anandjakhaniya) [inline backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java]: `unregisteredBackgroundUtilsCount` should be in common

File: backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java (hunk 2 above)
Comment (evgeniilezhnin) [reply backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java:682]: moved

File: backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java (hunk 3 above)
Comment (evgeniilezhnin) [reply backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java]: moved

File: backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java (hunk 4 above)
Comment (evgeniilezhnin) [reply backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java]: moved

**Review by anandjakhaniya — APPROVED** [review]
LGTM

**Review by conradoverta — APPROVED** [review]

**Pull request merged**